site/src/components/Footer.tsx

```diff
@@ -87,38 +83,67 @@ const SOCIALS: { name: string; icon: ReactNode; href: string }[] = [
     icon: <Icon className="fab fa-discord" />,
     href: "https://discord.gg/PefPteFe5j",
   },
+  {
+    name: "GitHub",
+    icon: <Icon className="fab fa-github" />,
+    href: "https://github.com/blockprotocol",
+  },
 ];
 
 const Socials = (
-  <Box mt={3} display="flex">
-    {SOCIALS.map(({ href, icon }) => (
-      <Link
-        href={href}
-        key={href}
-        sx={{
-          "&:not(:first-child)": {
-            marginLeft: 2.5,
-          },
-          ":hover": {
-            svg: {
-              color: (theme) => theme.palette.gray[30],
+  <Box
+    mt={3}
+    display="flex"
+    sx={{
+      flexDirection: {
+        xs: "column",
+        md: "row",
+      },
+      alignItems: { xs: "left", md: "center" },
+    }}
+  >
+    <Box marginBottom="1rem">
```

Comment:
Just curious, is there a reason for using `1rem` instead of the default theme spacing here?
```suggestion
    <Box marginBottom={2}>
```

Reply:
Have made this change in https://github.com/blockprotocol/blockprotocol/pull/75/commits/ae821f0ba94307366141bb6428de3c7c7abe9030